setup.py

```diff
@@ -127,6 +128,8 @@ def readfile(fname):
         Extension('gensim.models.doc2vec_inner',
             sources=['./gensim/models/doc2vec_inner.c'],
             include_dirs=[model_dir]),
+        Extension('gensim.pyemd.emd',
+            sources=['./gensim/pyemd/emd.cpp'])
```

Comment:
Please rely on pyEmd being installed from PyPi. Throw an exception in wmd if it is not installed.